horreum-backend/src/main/resources/application.properties

```diff
@@ -76,6 +88,7 @@ horreum.test-mode=false
 # thread pool sizes
 smallrye.messaging.worker.horreum.dataset.pool.max-concurrency=10
 smallrye.messaging.worker.horreum.run.pool.max-concurrency=10
+smallrye.messaging.worker.horreum.schema.pool.max-concurrency=10
```

Comment:
This increases the total number of `max-concurrency` to 30 but `quarkus.datasource.jdbc.max-size=25`
Can this lead to connection starvation for the UI?

Reply:
Do you see an issue in increasing the number of `quarkus.datasource.jdbc.max-size`? Otherwise I think we can reduce the `smallrye.messaging.worker.horreum.schema.pool.max-concurrency` as I am not expecting many schemas changes happen in parallel

Reply:
I know Jesper said our db could handle 100 connections, but now we're not running it on a separate machine any longer. I would be temped to keep the maximum number of connections and reduce each worker pool to 7 or something?

Reply:
We need to pick sensible defaults to run OOTB on any machine, the **actual** configuration should be defined at runtime to match the size of the machine / environment it is running on. The default configuration should not be configured for our (current) production environment.

Reply:
As default configuration here I kept 7,7,7